### DISCUSSION TITLE
JS backend script to mirror GitHub gists into note tree

### DISCUSSION THREAD

**Discussion by jwhonce** [TriliumNext/Trilium#4082] [Show and tell]
More fun in JavaScript, this copies gists from GitHub into a note tree. It is **NOT**  a sync tool. Content is pulled from GitHub and written into the trilium note subtree. This allows me to @link to a gist from a note and search gist content within trilium. Maybe someone else will find it useful. The script does not delete notes but it does rewrite content based on the fist filename as the tirilium note title.

 https://github.com/jwhonce/trilium-addons/blob/main/gistMirror/gistMirror.js

**Comment by meichthys** [comment]
Nice!

**Reply by meichthys** [reply]
Added to https://github.com/Nriver/awesome-trilium
